Documentation/k3d.md

```diff
@@ -18,7 +18,17 @@ Run the reset-local-k3d script, that automatically deletes the existing k3d clus
 ./Environment/k3d/reset-local-k3d
 ```
 
-For manual steps, continue following the guide.
+## Start the cluster
+> NOTE: This requires the cluster to be setup in order to work. Follow either the manual steps or run reset-local-k3d
+```sh
+./Environment/k3d/start-local-k3d
```

Comment:
```suggestion
k3d cluster start dolittle-dev
```

We actually only need one line in the script, the `k3d` command works globally so we don't have to `cd` anywhere. So I don't think that you even need the separate script files, just documenting the `k3d cluster start/stop dolittle-dev` is enough.